packages/app/src/debug/LayeredBrowserIcons.vue

```diff
@@ -57,7 +57,10 @@ const results = computed(() => {
 const BROWSER_MAP: Record<BrowserType, any> = {
   'CHROME': IconBrowserChrome,
   'CHROME BETA': IconBrowserChromeBeta,
-  'CHROME-CANARY': IconBrowserChromeCanary,
+  'CANARY': IconBrowserChromeCanary,
+  'CHROME CANARY': IconBrowserChromeCanary,
```

Comment:
Why canary and chrome canary?

Reply:
Chrome Canary is currently called 'Canary' in the app, however, this PR updates it to `Chrome Canary` but we need to handle both here.